### ISSUE TITLE
Move to new Replicated Files Extension API

### ISSUE THREAD

**Issue by carloscabanero** [blinksh/blink#1717]
The new API fixes all the problems we have previously seen with the previous and it is closer to the ones we have already implemented in other parts of the app.

The current Files.app extension is good enough for most use cases and should continue working. We would like to know how many of you are experiencing issues with it so we can prioritize how urgent the move to the new API is. 

So please leave your comments or thumbs up here!

**Comment by shlevy** [comment]
I'm not able to set a blink-exported location as the default download location for Safari

**Comment by carloscabanero** [comment]
Yes, this should come through with that.

**Comment by jimmy-zhening-luo** [comment]
Is the expected behavior that "link-files" does not persist between sessions?

**Comment by carloscabanero** [comment]
This is now a priority bc the issues with iOS18.

**Comment by shlevy** [comment]
Let me know if I can help test

**Comment by carloscabanero** [comment]
Update on this https://github.com/blinksh/blink/issues/2051#issuecomment-2429676502

**Comment by jimmy-zhening-luo** [comment]
> Is the expected behavior that "link-files" does not persist between sessions?

Ping on this, as this is not answered or documented anywhere.

**Comment by carloscabanero** [comment]
Yes, "link-files" are not persisted between sessions. I think this was suggested back in the day but we couldn't do it because they require explicit permission from the user every time. If you have observed that other apps are now able to do so, or if you know that maybe the APIs have changed, please feel free to open an Idea within [Discussions](https://github.com/blinksh/blink/discussions).

**Comment by carloscabanero** [comment]
Now on TestFlight 18.1.0

https://github.com/blinksh/blink/discussions/2067
